addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZwaveAssociationGroupInfoCommandClass.java

```diff
@@ -0,0 +1,397 @@
+ */
+package org.openhab.binding.zwave.internal.protocol.commandclass;
+
+import java.io.ByteArrayOutputStream;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.openhab.binding.zwave.internal.protocol.SerialMessage;
+import org.openhab.binding.zwave.internal.protocol.SerialMessage.SerialMessageClass;
+import org.openhab.binding.zwave.internal.protocol.SerialMessage.SerialMessagePriority;
+import org.openhab.binding.zwave.internal.protocol.SerialMessage.SerialMessageType;
+import org.openhab.binding.zwave.internal.protocol.ZWaveController;
+import org.openhab.binding.zwave.internal.protocol.ZWaveEndpoint;
+import org.openhab.binding.zwave.internal.protocol.ZWaveNode;
+import org.openhab.binding.zwave.internal.protocol.ZWaveSerialMessageException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamOmitField;
+
+/**
+ * Handles the Association Group Info Command command class.
+ *
+ * @author Jorg de Jong
+ */
+@XStreamAlias("associationGroupInfoCommandClass")
+public class ZwaveAssociationGroupInfoCommandClass extends ZWaveCommandClass
+        implements ZWaveCommandClassInitialization {
+
+    @XStreamOmitField
+    private static final Logger logger = LoggerFactory.getLogger(ZwaveAssociationGroupInfoCommandClass.class);
+
+    private static final byte GROUP_NAME_GET = 0x01;
+    private static final byte GROUP_NAME_REPORT = 0x02;
+    private static final byte INFO_GET = 0x03;
+    private static final byte INFO_REPORT = 0x04;
+    private static final byte COMMAND_LIST_GET = 0x05;
+    private static final byte COMMAND_LIST_REPORT = 0x06;
+
+    private static final int MAX_STRING_LENGTH = 25;
+
+    private static final int GET_LISTMODE_MASK = 0x40;
+    // not used private static final int GET_REFRESHCACHE_MASK = 0x80;
+
+    private static final int REPORT_GROUPCOUNT_MASK = 0x3f;
+    private static final int REPORT_DYNAMICINFO_MASK = 0x40;
+    private static final int REPORT_LISTMODE_MASK = 0x80;
+
+    private static final byte PROFILE_GENERAL = 0x00;
+    // general sub profile
+    private static final byte PROFILE_LIFELINE = 0x01;
+
+    // list of groups that the controller should subscribe to.
+    private Set<Integer> autoSubscribeGroups = new HashSet<>();;
+    private Map<Integer, GroupInfo> groups = new HashMap<>();
+
+    // list of command classes that are eligible for auto subscription.
+    @XStreamOmitField
+    private Set<CommandClass> autoCCs = ImmutableSet.of(CommandClass.DEVICE_RESET_LOCALLY, CommandClass.BATTERY,
+            CommandClass.CONFIGURATION);
```

Comment:
I added this because I saw it being reported on a lifeline group (the ZW098 led bulb)
It feels more like a 'strage/mis' configured device than any thing else, but I could also not see that the controller should not be subscribed.
An other candidate could maybe be  CENTRAL_SCENE?